Source/WPEFramework/PluginServer.h

```diff
@@ -2513,6 +2575,21 @@ namespace PluginHost {
             ~Channel() override;
 
         public:
+#ifdef THUNDER_CRASH_HANDLER
+            void GetRequestString(const Core::JSONRPC::Message &message, std::string& jsonRPCString)
```

Comment:
The concept behind thinder it to have a portable piece of code. WIth the change of a build flag we can turn in from char <->wchar. to do so we made a Portability.h where the strin is declared. Depending on the buildflag string -> std::string or std::wstring. By declaring std::string this feature is broken. Never use the std::string but always use string.